.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - "develop"
+      - "master"
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Incorporate Benchmark Data
+jobs:
+  benchmark-persist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+        # TODO: once the job is working, build semgrep-core from source
+      - name: Install semgrep-core from develop
+        run: sudo -E ./install-scripts/latest-artifact-for-branch.py
+        env:
+          SEMGREP_CORE: "y"
```

Comment:
REBUILD_SEMGREP_CORE might be more helpful

Reply:
it doesn't rebuild semgrep core - this is just a janky way of telling it what asset to download. 

semgrep-core from source coming later.